db/migrations/00029_create_header_sync_logs_table.sql

```diff
@@ -9,7 +9,7 @@ CREATE TABLE header_sync_logs
     data         BYTEA,
     block_number BIGINT,
     block_hash   VARCHAR(66),
-    tx_hash      VARCHAR(66),
+    tx_hash      VARCHAR(66) REFERENCES header_sync_transactions (hash),
```

Comment:
One thing to double check here is whether we may run into weird issues around the presence/absence of a `0x` prefix on the tx hash. I would think that we could expect the tx hashes we're seeing as uniform in that regard, but am hesitant do depend on that